packages/containers/src/EditableText/EditableText.test.js

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { Map } from 'immutable';
+import Container, { DISPLAY_NAME } from './EditableText.container';
+import Connect from './EditableText.connect';
+import { getEditMode } from './EditableText.selectors';
+
+describe('Connect', () => {
```

Comment:
I might be wrong, but I don't feel that this has any testing purpose around the current container